tests/js/HappinessLintBearTest.py

```diff
@@ -17,3 +26,23 @@
 HappinessLintBearTest = verify_local_bear(HappinessLintBear,
                                           valid_files=(good_file,),
                                           invalid_files=(bad_file,))
+
+
+@generate_skip_decorator(HappinessLintBear)
+class HappinessLintBearConfigTest(LocalBearTestHelper):
+
+    USE_SPACE_ERR_MSG = re.escape('"use_spaces=True" is incompatible with '
+                                  'HappinessLintBear, set it to false.')
+
+    def test_validate_use_spaces(self):
+        section = Section('name')
+        section.append(Setting('use_spaces', True))
+        bear = HappinessLintBear(section, Queue())
+
+        with self.assertRaisesRegex(AssertionError, self.USE_SPACE_ERR_MSG):
+            content = bad_file.splitlines()
+            with prepare_file(content, None) as (file, fname):
+                with execute_bear(bear, fname, file) as results:
+                    fdict = {fname: file}
+                    results[0].apply(fdict)
+                    self.assertEqual(''.join(fdict[fname]), good_file)
```

Comment:
you have read my comment right? You just need to do `bear.execute` below the `with` statement. You don't need `prepare_file` or any other function you have here.

Reply:
![image](https://user-images.githubusercontent.com/25884863/33565675-ceb3270a-d943-11e7-8297-1d549294b02c.png)
This is what you are telling right?

Reply:
wait theres no need of splitting the lines as well i suppose.